src/modules/Api.js

```diff
@@ -0,0 +1,39 @@
+const url = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/0pNrfjBwCKXqbjF87mV/scores/';
+const addData = function (name, score) {
```

Comment:
[REQUIRED] It is a requirement in this milestone for you to use arrow functions instead of the function keyword. Kindly fix this.